Library/Tracking/KSRAnalyticsTests.swift

```diff
@@ -2340,8 +2340,8 @@ final class KSRAnalyticsTests: TestCase {
     XCTAssertEqual("2016-09-16T22:35:15Z", props?["project_launched_at"] as? String)
     XCTAssertEqual("live", props?["project_state"] as? String)
     XCTAssertEqual(1_000, props?["project_current_pledge_amount"] as? Int)
-    XCTAssertEqual(1_000, props?["project_current_amount_pledged_usd"] as? Float)
-    XCTAssertEqual(2_000, props?["project_goal_usd"] as? Float)
+    XCTAssertEqual(2_000, props?["project_current_amount_pledged_usd"] as? Int)
```

Comment:
The changes in the values of these two properties is quite significant. Can you explain?

Reply:
If you check the stats property of the `Project.template` which `Project.Stats.template`, you will notice that `convertedPledgedAmount` is 2_000 which is the new value use for `project_current_amount_pledged_usd` property.
For `goal_usd` the new value is `goalCurrentCurrency` which is the product of `currentCurrencyRate` and `goal` amount. That will give you 3_000.